src/VisualStudio/Core/Def/RoslynPackage.cs

```diff
@@ -123,11 +126,18 @@ protected override async Task LoadComponentsAsync(CancellationToken cancellation
             // the appropriate task scheduler to report events on.
             this.ComponentModel.GetService<MiscellaneousFilesWorkspace>();
 
-            LoadAnalyzerNodeComponents();
+            LoadAnalyzerNodeComponents();            
 
             LoadComponentsBackgroundAsync(cancellationToken).Forget();
         }
 
+        private void RegisterCodeCleanupProvider()
+        {
+            var regService = ComponentModel.GetService<ICodeCleanUpFixerRegistrationService>();
```

Comment:
hmm... this seems little bit weird. since this will load any provider in VS (I mean dlls)

Reply:
➡️ We load the fixer registration service, but only load Roslyn's fixer implementation.